packages/mock-addon-docs/package.json

```
@@ -15,20 +15,21 @@
   "author": "Farhad Yasir(nutboltu)",
   "license": "MIT",
   "devDependencies": {
-    "@storybook/addon-actions": "^7.4.2",
-    "@storybook/addon-essentials": "^7.4.2",
-    "@storybook/addon-interactions": "^7.4.2",
-    "@storybook/addon-links": "^7.4.2",
-    "@storybook/react-webpack5": "^7.4.2",
+    "@storybook/addon-actions": "8.0.5",
+    "@storybook/addon-essentials": "^8.0.5",
+    "@storybook/addon-interactions": "8.0.5",
+    "@storybook/addon-links": "8.0.5",
+    "@storybook/react": "^8.0.6",
+    "@storybook/react-vite": "^8.0.6",
     "@storybook/storybook-deployer": "^2.8.16",
-    "storybook": "^7.4.2"
+    "storybook": "8.0.5"
```

I'd recommend to use caret so:
```suggestion
    "storybook": "^8.0.5"
```